planning/behavior_path_planner/autoware_behavior_path_planner_common/include/autoware/behavior_path_planner_common/utils/path_safety_checker/objects_filtering.hpp

```diff
@@ -38,10 +38,14 @@ bool velocity_filter(
   const PredictedObject & object, double velocity_threshold, double max_velocity);
 
 bool position_filter(
-  PredictedObject & object, const std::vector<PathPointWithLaneId> & path_points,
+  const PredictedObject & object, const std::vector<PathPointWithLaneId> & path_points,
   const geometry_msgs::msg::Point & current_pose, const double forward_distance,
   const double backward_distance);
 
+bool is_within_circle(
```

Comment:
can you add doxygen comments for the tested functions?

Reply:
Added in 2870c6bee3e9a468824f29b77efe5f3f6a4e9077, except for already existing ones